org.cosmic.Wam.json

```diff
@@ -0,0 +1,53 @@
+{
+    "app-id": "org.cosmic.Wam",
```

Comment:
cosmic.org says 404 not found

Reply:
Oh well. I don't own that domain. So it should point to my GH then?

Reply:
Yes see the second last checklist item. If you are using the `org.cosmic.Wam` id in your codebase those needs to change too.

Reply:
Ok so this I will change upstream and then I need to update this PR, ok I got it. Sorry for confusions it's my first time with this kind of job here ;p

Reply:
Before submit a commit I will ask if now, [io.github.elevenhsoft.WebApps](https://github.com/elevenhsoft/cosmic-wam/blob/master/io.github.elevenhsoft.WebApps.json) do the job?

Reply:
I changed in codebase also tested locally and everything seems to works correctly with new id.

Reply:
I'd suggest making the last part cosmic-wam to match the repo name.

Reply:
Ok, sure. I can rename it for `webapps`. Also will change to upstream repo name to WebApps from `elevenhsoft/cosmic-wam` .

Reply:
Done. Please check now my friend: https://github.com/elevenhsoft/WebApps

Reply:
Seems fine, you have to rename things in this PR now.

Reply:
Ok, awesome. Thank you for your assistance. I will do a commit for a moment with new manifest and cargo-sources.json. I will need help also with this exception right above. But I will ask there ^^